### PR TITLE
integrate info from stubbed starburst-side doc

### PR DESCRIPTION
**Pull request by dataders**
## What are you changing in this pull request and why?

Add information that was previously on [Starburst doc's dbt page
](https://docs.starburst.io/data-consumer/clients/dbt.html)

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/reference/resource-configs/trino-configs.md [hunk 1]
```diff
@@ -3,11 +3,17 @@ title: "Starburst/Trino configurations"
 id: "trino-configs"
 ---
 
+## Cluster requirements
+
+The designated cluster must have an attached catalog where ojects such as tables and views can be created, renamed, altered, and dropped. And, any user connecting to the cluster with dbt must also have these same permissions for the target catalog.
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: objects

Comment (mosabua) [reply website/docs/reference/resource-configs/trino-configs.md]: Drop the "And,"

File: website/docs/reference/resource-configs/trino-configs.md [hunk 2]
```diff
@@ -3,11 +3,17 @@ title: "Starburst/Trino configurations"
...
+
 ## Session properties
 
-In the Starburst/Trino environment, you can set session properties that temporarily modifies the current session you have with the cluster. Refer to [Set Session](https://docs.starburst.io/latest/sql/set-session.html) in the Starburst/Trino docs for more details.
+In the Starburst/Trino environment, you can set session properties that modify the current session you have with the cluster. Refer to [Set Session](https://docs.starburst.io/latest/sql/set-session.html) in the Starburst/Trino docs for more details.
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]:
```
With a Starburst Enterprise, Starburst Galaxy, or Trino cluster, you can [set session properties](https://trino.io/docs/current/sql/set-session.html) to modify the current configuration for your user session.
```

File: website/docs/reference/resource-configs/trino-configs.md [hunk 3]
```diff
@@ -3,11 +3,17 @@ title: "Starburst/Trino configurations"
...
+
+The standard way to define session properties is via the `session_properties` field of your `profiles.yml`. This ensures that all dbt connections use these settings by default.
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: is with the

Comment (mosabua) [reply website/docs/reference/resource-configs/trino-configs.md]: Thats also a problem btw.. it affects all connections .. and therefore potentially different cluster ... not sure if it makes sense to use this as a general recommendation. wdyt @hovaesco ?

File: website/docs/reference/resource-configs/trino-configs.md [hunk 4]
```diff
@@ -3,11 +3,17 @@ title: "Starburst/Trino configurations"
...
 
-During a dbt connection session, you can use [dbt hooks](/reference/resource-configs/pre-hook-post-hook) to set Starburst/Trino session properties on a specific dbt model. For example: 
+However, to temporaily adjust these session properties for a specific dbt model or group of models, you can use a [dbt hook](/reference/resource-configs/pre-hook-post-hook) to set Starburst/Trino session properties on a specific dbt model. For example:
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: drop the `Starburst/Trino` .. its redundant here

File: website/docs/reference/resource-configs/trino-configs.md [hunk 5]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
 
 For details on what's supported for each supported data source, refer to either the [Trino Connectors](https://trino.io/docs/current/connector.html) or [Starburst Catalog](https://docs.starburst.io/starburst-galaxy/catalogs/).
 
-For example, this configures the data to materialize as a [Parquet](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html) table with partitions: 
+
+
+### Hive Catalog
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
### Hive catalogs
```

File: website/docs/reference/resource-configs/trino-configs.md [hunk 6]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
...
+
+At target cataog that uses the Hive connector and a metastore service (HMS) is very typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
+
+```java
+hive.metastore-cache-ttl=0s
+hive.metastore-refresh-interval=5s
+hive.allow-drop-table=true
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: this is no longer true

File: website/docs/reference/resource-configs/trino-configs.md [hunk 7]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
...
+hive.allow-rename-table=true
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: this is no longer true

File: website/docs/reference/resource-configs/trino-configs.md [hunk 8]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
 
 For details on what's supported for each supported data source, refer to either the [Trino Connectors](https://trino.io/docs/current/connector.html) or [Starburst Catalog](https://docs.starburst.io/starburst-galaxy/catalogs/).
 
-For example, this configures the data to materialize as a [Parquet](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html) table with partitions: 
+
+
+### Hive Catalog
+
+At target cataog that uses the Hive connector and a metastore service (HMS) is very typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: catalog

Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: Drop "very"

Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: Last sentence should be removed

File: website/docs/reference/resource-configs/trino-configs.md [hunk 9]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
...
+```java
+hive.metastore-cache-ttl=0s
+hive.metastore-refresh-interval=5s
+hive.allow-drop-table=true
+hive.allow-rename-table=true
+```
+
+## File format configuration
+
+When using file-based connectors such as Hive, a user can customize how to use certain
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: Sentence is incomplete .. also not sure what you want to say..

File: website/docs/reference/resource-configs/trino-configs.md [hunk 10]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
...
+hive.metastore-refresh-interval=5s
+hive.allow-drop-table=true
+hive.allow-rename-table=true
+```
+
+## File format configuration
+
+When using file-based connectors such as Hive, a user can customize how to use certain 
+
+The below configures the data to materialize as a [Parquet](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html) table with partitions.
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: There is no such thing as a Parquet table .. its a Parquet file.. or files in this context

File: website/docs/reference/resource-configs/trino-configs.md [hunk 11]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
...
+At target cataog that uses the Hive connector and a metastore service (HMS) is very typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
+
+```java
+hive.metastore-cache-ttl=0s
+hive.metastore-refresh-interval=5s
+hive.allow-drop-table=true
+hive.allow-rename-table=true
+```
+
+## File format configuration
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md:43]: This should be made more generic and talk about table properties probably

File: website/docs/reference/resource-configs/trino-configs.md (hunk 8 above)
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md]: Also .. this should probably be expanded to talk about Hive, Iceberg, and Delta Lake

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
Please ping me when this is closer to merge @dataders .. I have a bunch of other edits for that page but I want to avoid having to deal with the conflicts

File: website/docs/reference/resource-configs/trino-configs.md (hunk 3 above)
Comment (hovaesco) [reply website/docs/reference/resource-configs/trino-configs.md]: The general recommendation should be to setup as much as possible on the server side. Session properties would be only required in the specific use cases (not common I'd say).

**Comment by mosabua** [comment]
@nghi-ly and @dataders .. are you proceeding with this PR?

File: website/docs/reference/resource-configs/trino-configs.md (hunk 1 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
The designated cluster must have an attached catalog where objects such as tables and views can be created, renamed, altered, and dropped. Any user connecting to the cluster with dbt must also have these same permissions for the target catalog.
```

File: website/docs/reference/resource-configs/trino-configs.md (hunk 2 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
With a Starburst Enterprise, Starburst Galaxy, or Trino cluster, you can [set session properties](https://trino.io/docs/current/sql/set-session.html) to modify the current configuration for your user session.
```

File: website/docs/reference/resource-configs/trino-configs.md (hunk 3 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
The standard way to define session properties is with the `session_properties` field of your `profiles.yml`. This ensures that all dbt connections use these settings by default.
```

File: website/docs/reference/resource-configs/trino-configs.md (hunk 4 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
However, to temporaily adjust these session properties for a specific dbt model or group of models, you can use a [dbt hook](/reference/resource-configs/pre-hook-post-hook) to set session properties on a specific dbt model. For example:
```

File: website/docs/reference/resource-configs/trino-configs.md (hunk 8 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]: why?

Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
At target catalog that uses the Hive connector and a metastore service (HMS) is typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
```

Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]: can this be addressed in a future PR?

File: website/docs/reference/resource-configs/trino-configs.md (hunk 6 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]: what's the suggestion? I copied and pasted this from your `dbt.md`

File: website/docs/reference/resource-configs/trino-configs.md (hunk 7 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]: what's the suggestion? I copied and pasted this from your `dbt.md`

File: website/docs/reference/resource-configs/trino-configs.md (hunk 9 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
When using file-based connectors such as Hive, a user can customize aspects of the connector such as the format that is used as well the type of materialization
```

File: website/docs/reference/resource-configs/trino-configs.md (hunk 10 above)
Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
The below configures the table to be materializes as a set of partitioned [Parquet](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html) files.
```

**Review by dataders — COMMENTED** [review]
@mosabua tackled your low-hanging edits. The following suggestions I think should happen in a future PR so as to unblock you:
- Last sentence should be removed (https://github.com/dbt-labs/docs.getdbt.com/pull/3233/files#r1173155452)
- Also .. this should probably be expanded to talk about Hive, Iceberg, and Delta Lake (https://github.com/dbt-labs/docs.getdbt.com/pull/3233/files#r1173156664)
- This should be made more generic and talk about table properties probably (https://github.com/dbt-labs/docs.getdbt.com/pull/3233/files#r1173156387)

File: website/docs/reference/resource-configs/trino-configs.md (hunk 7 above)
Comment (mosabua) [reply website/docs/reference/resource-configs/trino-configs.md]: Remove .. that property is no longer exists

File: website/docs/reference/resource-configs/trino-configs.md (hunk 6 above)
Comment (mosabua) [reply website/docs/reference/resource-configs/trino-configs.md]: Remove .. that property is no longer exists

**Review by mosabua — DISMISSED** [review]
Two minor changes needed .. then good to go

File: website/docs/reference/resource-configs/trino-configs.md [hunk 12]
```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
...
+
+### Hive catalogs
+
+At target catalog that uses the Hive connector and a metastore service (HMS) is typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
+
+```java
+hive.metastore-cache-ttl=0s
+hive.metastore-refresh-interval=5s
+hive.allow-drop-table=true
+hive.allow-rename-table=true
```
Comment (dataders) [inline website/docs/reference/resource-configs/trino-configs.md]:
```suggestion
```

File: website/docs/reference/resource-configs/trino-configs.md (hunk 8 above)
Comment (mosabua) [reply website/docs/reference/resource-configs/trino-configs.md]: Because the properties are no longer there .. and this no longer applies.

**Review by runleonarun — APPROVED** [review]
Looks ready to ship!

File: website/docs/reference/resource-configs/trino-configs.md [hunk 13]
```diff
@@ -23,7 +29,22 @@ You can use Starburst/Trino table properties to configure how you want your data
 
 For details on what's supported for each supported data source, refer to either the [Trino Connectors](https://trino.io/docs/current/connector.html) or [Starburst Catalog](https://docs.starburst.io/starburst-galaxy/catalogs/).
 
-For example, this configures the data to materialize as a [Parquet](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html) table with partitions: 
+
+
+### Hive catalogs
+
+At target catalog that uses the Hive connector and a metastore service (HMS) is typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
```
Comment (mosabua) [inline website/docs/reference/resource-configs/trino-configs.md:36]: The last sentence still needs to go .. but then this looks ready

Comment (dataders) [reply website/docs/reference/resource-configs/trino-configs.md:36]: why?

**Pull request merged**